partition/roles/ztp/templates/ztp.json.j2

```diff
@@ -5,17 +5,35 @@
         "url": "http://{{ ztp_listen_address }}:{{ ztp_port }}/ztp.sh"
```

Comment:
What do you think about remaining it to `user.sh`?
```suggestion
        "url": "http://{{ ztp_listen_address }}:{{ ztp_port }}/user.sh"
```

Reply:
Done